stimuli_ci_generation/src/server/main.py

```diff
@@ -63,7 +66,8 @@ def index():
         else:
             try:
                 generate_stimuli(participant_id, file_name)
-                return ("Generating stimuli images...", 202)
+                pub_msg("Stimuli generated", SIE_IMG_PROCESSING_RESULT, participant_id)
```

Comment:
refer to the design doc for the message format.

Format for folder names and messages:
[username]-[session-id/experiment-id]-[status]
Status: “complete”, “face_missing”, “failed”
Example: “username-001-complete”

Reply:
ah gotcha, I should have looked there first! Thanks!

By `username`, you mean `participant_id` right?

Reply:
Actually, I remember the firebase team decided to use the username rather than ID, they think the ID is too long.

Reply:
I think the backend is using username-sessionid to track users and the specific survey session? but all we need to care about is the whole identifier [username]-[session-id/experiment-id]

Reply:
> Actually, I remember the firebase team decided to use the username rather than ID, they think the ID is too long.

doesn't matter what content they decide to use, it shouldn't affect us as long as the format is consistent.

Reply:
`username-experimentId` might not be unique though? I'm not sure what `sessionId` is?

Reply:
should we change all usage of `participant_id` to `username` then? To stay consistent? Also what is `sessionId`?

Reply:
For now, participant_id looks good to me, since the frontend team will pass the user_id to the imageService. Thus the participant_id you got in your side will be the user_id.

Reply:
For sessionId I think it's the experiment Id then, since during one self-image experiment, a participant will upload a photo once. Thus we connect user_id with experiment_id.

Reply:
okay, just to confirm `[participant_id]-[experiment_id]-[status]` it is

Reply:
We inherited participant_id from the previous repo, in my opinion a more appropriate name should be just identifier

Reply:
Addressed your comments but with a json formatted message. Basically message will be `<status>` but you can extract `participant_id` and `experiment_id` from the json object. This way you won't need to do string processing. What do you guys think?

Reply:
> okay, just to confirm `[participant_id]-[experiment_id]-[status]` it is

Should be `[participant_id]-[status]` for us

Reply:
Yep no problems with/without `experiment_id`. Btw to clarify, this is what the json message will look like:
```
publisher.publish(
        topic_path,
        data,  # actuall message .i.e. status
        participant_id=participant_id,
        experiment_id=experiment_id,
        retry=custom_retry,
)
```

Reply:
Lgtm, well be fine as long as the fire base team knows what we are doing

Reply:
LGTM, the experiment id will always be provided when we do final testing/production, but for now no problem.